layers/+lang/python/funcs.el

```diff
@@ -56,12 +56,6 @@
   (forward-button 1)
   (call-interactively #'push-button))
 
-(defun spacemacs//disable-semantic-idle-summary-mode ()
```

Comment:
Since the `spacemacs//disable-semantic-idle-summary-mode` function is being moved from the `python` layer to the `semantic` layer, and it's still being called in the python layers `python/post-init-semantic` function.

Maybe it needs to check that the `semantic` layer is being used before adding the hook.

https://github.com/syl20bnr/spacemacs/blob/5d02b1c4bba2ce5b08c20731598d411c09a8acc1/layers/%2Blang/python/packages.el#L376-L377

Reply:
`python/post-init-semantic` will only be called when both `python` layer and `semantic` layer are enabled, which is handled by `spacemacs` automatically.